allensdk/model/biophysical/runner.py

```diff
@@ -212,8 +227,12 @@ def load_description(manifest_json_path):
     return description
 
 
-if '__main__' == __name__:
-    import sys
+# Create the parser
+sim_parser = argparse.ArgumentParser(description='Run simulation for biophysical models with the provided configuration')
+sim_parser.add_argument('manifest_file',
+                        help='.json configurations for running the simulations')
+sim_parser.add_argument('--axon_type', required=False, help='axon replacement for biophysical models')
```

Comment:
better to create a default value to the argument, thus the axon_type will always have the value obviating a need for handling cases when the value does not exist. Also using `choices` option is probably better as it allows limiting the values of parameters to a particular set of know choices:
e.g.:
choices=['stub', 'truncated']
use the `help` argument to describe here what each choice of axon_type does to the reconstruction.

Reply:
@sgratiy 
I can't add a default value since the default for legacy perisomatic (stub) and all-active models (truncated) are different and we'd have to change how these models were run previously. I am trying to keep backward compatibility. Will add the choices though.
Thanks

Reply:
Maybe I can just specify in the help message that these option would only be limited to the all-active models. Would it be acceptable?

Reply:
Sure, just specify the help message. It is acceptable.

Reply:
Hi @sgratiy 
Modified the code with the default axon replacement for the models (specified in the help message).
Thanks